bigtable/admin/instance_admin.h

```diff
@@ -0,0 +1,82 @@
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_BIGTABLE_ADMIN_INSTANCE_ADMIN_H_
+#define GOOGLE_CLOUD_CPP_BIGTABLE_ADMIN_INSTANCE_ADMIN_H_
+
+#include <memory>
+#include "bigtable/admin/instance_admin_client.h"
+#include "bigtable/client/internal/unary_rpc_utils.h"
+
+namespace bigtable {
+inline namespace BIGTABLE_CLIENT_NS {
+/**
+ * Implements a minimal API to administer Cloud Bigtable instances.
+ */
+class InstanceAdmin {
```

Comment:
Does this class need to be copyable?

Reply:
No, but there is no reason for it to *not* be copyable either.

Reply:
In that case please lock it down. Making things non copyable nudges people to think harder about the ownership model.

Reply:
Done.